Basic-Recruit-Training/main.cpp

```diff
@@ -19,10 +19,16 @@ void compareAnswers(const std::string myAnswer, const std::string correctAnswer)
 // Swaps a & b.
 void mySwap(int& a, int& b)
 {
+    int temp=b;
+    a=b;
+    b=temp;
+    return a,b
```

Comment:
This is a void function, returning these arguments doesn't do anything. C++ also does not support returning multiple values like this (I think that was the intent), so actually I don't think this compiles?